sdk/core/azure-core/src/main/java/com/azure/core/implementation/util/FluxByteBufferContent.java

```diff
@@ -91,4 +91,11 @@ public ByteBuffer toByteBuffer() {
                 .share()
                 .block();
     }
+
+    @Override
+    public BinaryDataContent copy() {
+        // TODO (alzimmer): What does it mean to copy a Flux<ByteBuffer>?
```

Comment:
The `HttpRequest copy()` didn't really copy the body before. I think instead of copy we may think how one could provide a supplier of flux to httprequest ctor so that content can be replayed. 
For example maybe we need a marker "ReplyableContent" which can be right a way aplied to StringContent and ByteArrayContent, but Flux and Stream would need subtypes.